aea/protocols/__init__.py

```diff
@@ -18,3 +18,10 @@
 # ------------------------------------------------------------------------------
 
 """This module contains the protocol modules."""
+from typing import List
+
+default_dependencies = []  # type: List[str]
+fipa_dependencies = ["protobuf"]  # type: List[str]
+gym_dependencies = []  # type: List[str]
```

Comment:
We could add `gym` here and then replace the `gym_Env` stub with `gym.Env`. Thoughts?

Reply:
yes, agreed!